generateTemplat.sh

```diff
@@ -10,7 +10,10 @@ if [ "$1" != "" ]; then
     mkdir $1
     cd $1
     echo "{ \"lesson_title\": \"$1\" }" >> lesson_settings.json
+    echo "## $1" >> lesson.md
```

Comment:
this file has a typo in the name, also we're doing kebab-case.

Reply:
we don't get to choose the case, its based on repl.it requirements.